public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -16,7 +16,7 @@ block includes
   [JSONP](https://en.wikipedia.org/wiki/JSONP). A few browsers also support
   [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). 
   
-  The Angular HTTP client library simplifies application programming of the **XHR** and **JSONP** APIs
+  The !{_Angular_http_library} simplifies application programming of the **XHR** and **JSONP** APIs
```

Comment:
This link doesn't work in the Dart version. It goes to: `http://localhost:3000/docs/dart/latest/guide/!%7B_httpUrl%7D`